conf/openmetadata.yaml

```diff
@@ -156,6 +156,10 @@ database:
   evictionInterval: ${DB_CONNECTION_EVICTION_INTERVAL:-5 minutes}
   minIdleTime: ${DB_CONNECTION_MIN_IDLE_TIME:-1 minute}
 
+objectStorage:
+  enabled: false
+  provider: NOOP
```

Comment:
```suggestion
  provider: NOOP # Supported values - "s3", "azure", or "noop" (for local testing)
```

Reply:
its meant to be no configurable in case of OSS